frontend/src/Components/Common/EditButton.js

```diff
@@ -0,0 +1,8 @@
+export function EditButton(props) {
+    return (
+        <button onClick={props.onClick} type="button" className="btn btn-outline-primary">
```

Comment:
E doar o sugestie -- banuiesc ca aceste clase incarca CSS din bootstrap.

Ce as fi facut eu diferit -- as fi ales o librarie de componente deja existenta pentu react, te ajuta sa nu iti pierzi timpul cu unele detalii. Recomand material-ui.

Metodele moderne de CSS din react sunt bazate pe scoped CSS, styled-components, emotion sau JSS.

Reply:
Voi schimba libraria de UI din Bootstrap, in Material-UI. Implicit, aceasta observatie cat si celelalte care sunt adiacente subiectului, vor fi rezolvate.